### ISSUE TITLE
Adguard and vpn

### ISSUE THREAD

**Issue by blogdemoi** [YunoHost-Apps/adguardhome_ynh#58]
Hi,

I have a Yunohost installation + a vpn provided by the FFDN, using the Vpn Yunohost App. Can I use Adguard for Yunohost with this configuration?

Thank you.

**Comment by Optogram** [comment]
I'm using Yunohost and a VPN and it's working without issues.
The only thing that seems strange is that I can't use my local IP as DNS, Adguard uses the external one provided by the VPN :/

**Comment by Ddataa** [comment]
hello @blogdemoi is this issue still valid ?

**Comment by blogdemoi** [comment]
> hello @blogdemoi is this issue still valid ?

Yes, I haven't tried to install Adguard as I am also using a Vpn via the vpnclient_ynh app https://github.com/YunoHost-Apps/vpnclient_ynh and I am afraid of breaking my setup.

**Comment by blogdemoi** [comment]
> I've tried again a few days ago, the result was the same. Adguard uses the external one provided by the VPN.

Thank you for letting me know, I am going to try then.

**Comment by Optogram** [comment]
You can use it, but everyone who knows your external VPN IP can use it as well, so this is an issue for me, as stated here : https://github.com/YunoHost-Apps/adguardhome_ynh/issues/72

I want to use Adguard locally, not for everyone on the internet :/

**Comment by Ddataa** [comment]
so we shall either close this issue or the #72 if they are related to the same problematic.

no ?

**Comment by Optogram** [comment]
They are not. @blogdemoi asked if Adguard could be used with the VPNClient App, and that's correct, it works.
But Adguard is using the external IP adress of the VPN , not the local IP adress. So, you're just exposing Adguard to the world, and you can't use it on your local network. To me, that's an issue :/

**Comment by Ddataa** [comment]
understood

so this issue here is solved then ?

**Comment by Optogram** [comment]
Yep ! :)

**Comment by blogdemoi** [comment]
> so this issue here is solved then ?

Yes, thank you.
